extras/src/main/java/com/google/gson/graph/GraphAdapterBuilder.java

```diff
@@ -51,6 +81,13 @@ public GraphAdapterBuilder() {
             instanceCreators, true, Collections.<ReflectionAccessFilter>emptyList());
   }
 
+  /**
+   * Registers the specified type with a default instance creator. The default instance creator is
+   * obtained using the {@link ConstructorConstructor} for the given type.
```

Comment:
I think I would delete this sentence, since `ConstructorConstructor` is an internal type.